sessions/react-usestate-and-events-advanced/lifting-up-state-personal-details/src/Form.js

```diff
@@ -0,0 +1,26 @@
+import {useState} from 'react';
+
+export default function Form({onUserCreate}) {
+	const [name, setName] = useState('');
+	const [email, setEmail] = useState('');
+
+	function handleSubmit(event) {
+		event.preventDefault();
+		const form = event.target;
+		const {name, email} = form.elements;
+		setName(name.value);
+		setEmail(email.value);
```

Comment:
Students might get confused by variable shadowing of `name` and `email`. I suggest using different names for the input fields.

```suggestion
		const {userName, userEmail} = form.elements;
		setName(userName.value);
		setEmail(userEmail.value);
```